schema/reportdb/uyuni-reportdb-schema.changes

```diff
@@ -1,3 +1,6 @@
+-------------------------------------------------------------------
```

Comment:
Here and in the other changelog, please remove this separator line, it is not needed. It will be automatically added when the new version is released.

Reply:
fixed